### PR TITLE
Target node plus

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
WIP, based off of the locked `targetNode` branch.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # did-veres-one ChangeLog
 
+## 4.0.0 - 2019-01-08
+
+### Changed
+- **BREAKING** Using the new flatter `Ed25519VerificationKey2018` type suites.
```
Comment (dlongley) [inline CHANGELOG.md]: Perhaps reword this to avoid confusion:

`Replace "cryptographic suites" with flat verification methods like `Ed25519VerificationKey2018`.

Comment (davidlehn) [inline CHANGELOG.md]: Style nit.  Renders nicer with a `:` IMHO.
```suggestion
- **BREAKING**: Using the new flatter `Ed25519VerificationKey2018` type suites.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+- **BREAKING** Removed injector usage, removed related `v1.use()` notation.
```
Comment (davidlehn) [inline CHANGELOG.md]:
```suggestion
- **BREAKING**: Removed injector usage, removed related `v1.use()` notation.
```

File: lib/VeresOne.js [hunk 3]
```diff
@@ -274,11 +277,23 @@ class VeresOne {
     this.logger.log('Sending to ledger, operation type:', operation.type);
     const {didDocument} = options;
 
+    let targetNode;
+    try {
+      targetNode = await targetNodeClient.getTargetNode(
+        {hostname: this.hostname});
+    } catch(e) {
+      console.log('Error in getTargetNode.', e.message);
```
Comment (davidlehn) [inline lib/VeresOne.js]: Does this need to be logged?  If so, better to use the `this.logger`?  (Does it have an `error()` method?)

File: CHANGELOG.md (hunk 1 above)
Comment (mattcollier) [reply CHANGELOG.md]: fixed.

Comment (mattcollier) [reply CHANGELOG.md]: Fixed.

File: lib/VeresOne.js (hunk 3 above)
Comment (mattcollier) [reply lib/VeresOne.js]: fixed.
